accounts/accounts-model.js

```diff
@@ -0,0 +1,44 @@
+const db = require("../data/dbConfig");
+
+function getAccounts(sort) {
+  if (sort) {
+    return db("accounts")
+      .orderBy(sort.sortby, sort.sortdir)
+      .limit(sort.limit);
+  } else {
+    return db("accounts");
+  }
+}
```

Comment:
Since the `sort` is optional you should initialize it to null like `function getAccounts(sort = null) ` else you would always need to pass `sort` to `getAccounts` call